### PR TITLE
Email invitations

### PR DESCRIPTION
**Pull request by dburka**
Story: https://www.pivotaltracker.com/story/show/171152569

New email invitation formatting for 2 key system emails from Simple Dashboard. We will design and push more emails over the next few weeks, but let's do this piecemeal. I have tested these in most modern email clients.

### CODE REVIEW

File: app/views/approval_notifier_mailer/registration_approval_email.html.erb [hunk 1]
```diff
@@ -1,41 +1,109 @@
...
+                <!--NORMAL BUTTON-->
+                    <a href="<%= admin_user_url(@user) %>" style="background-color:#FF3355;border:1px solid #c20020;border-radius:5px;color:#ffffff;display:inline-block;font-family:sans-serif;font-size:16px;font-weight:bold;line-height:42px;text-align:center;text-decoration:none;width:220px;-webkit-text-size-adjust:none;mso-hide:all;">
+                    <%= t("registration_approval_email.allow_or_deny_access") %>
+                    </a>
+            </div>
+        
+            <div>&nbsp;</div>
+            <div>&nbsp;</div>
+
+            <!--HR-->
```
Comment (harimohanraj89) [inline app/views/approval_notifier_mailer/registration_approval_email.html.erb]: Gotta love email markup 😄

**Comment by dburka** [comment]
I went ahead and formatted ALL of the major emails. But, I wasn't able to trigger them all to show up in Mailcatcher, so we should ensure I didn't goof anything. Help?

**Comment by harimohanraj89** [comment]
Also from DBurka:

> I am trying to format the rest of the emails we send from the Dashboard . I have Mailcatcher working correctly but I can’t figure out how to trigger:
> * email_changed (is this one even used? … what’s it for?)
> * confirmation_instructions
> * registration_approval_email.html.erb
> Also, where is the file for the email that comes when you download the Patient Line List for a district or a facility? It also doesn’t come up in Mailcatcher.

**Comment by harimohanraj89** [comment]
As discussed in Slack, `email_changed` and `confirmation_instructions` are not used.

`registration_approval_email.html.erb` screenshot attached below.

Screenshot 2020-03-02 at 11 09 32 AM

**Comment by harimohanraj89** [comment]
Moar screenshots

Screenshot 2020-03-02 at 11 21 05 AM

Screenshot 2020-03-02 at 11 22 35 AM

Screenshot 2020-03-02 at 11 35 15 AM

**Comment by kitallis** [comment]
These emails look like Real Deal™ now 👍

**Comment by harimohanraj89** [comment]
Much div, many nbsp, wow.

**Pull request merged**